app/src/services/graph.service.js

```diff
@@ -16,8 +17,9 @@ class GraphService {
                 }
             });
         } catch (e) {
-            logger.error(e);
-            throw new Error(e);
+            const errorMsg = '[GraphService]: Error communicating with Graph MS (POST associations)';
+            logger.error(errorMsg, e.message);
```

Comment:
I'd suggest removing the logger.error from here, and handling things here instead: https://github.com/resource-watch/vocabulary-tag/blob/develop/app/src/app.js#L85

that way, we have separation of concerns:
- the code generates the message, and the corresponding error code (in this case, a 500, as it's an error internal to our implementation, as opposed to, say, an invalid token provided by a user)
- the logger bit on app.js is responsible for deciding what to do with that error.

Reply:
Remove logger references from graph service, now just throwing an error with the correct error message.